containers/reasonml/README.md

```diff
@@ -0,0 +1,57 @@
+# ReasonML
+
+## Summary
+
+*Develop [reasonml](https://reasonml.github.io/en/) applications. Includes [fish shell](https://fishshell.com/).*
+
+| Metadata | Value |  
+|----------|-------|
+| *Contributors* | Diullei Gomes ([@diullei](https://github.com/diullei)) |
+| *Definition type* | Dockerfile |
+| *Languages, platforms* | ReasonML |
+
+## Using this definition with an existing folder
+
+This image contains the development environment to work with ReasonML applications. To get started, follow these steps:
+
+1. If this is your first time using a development container, please follow the [getting started steps](https://aka.ms/vscode-remote/containers/getting-started) to set up your machine.
+
+2. To use VS Code's copy of this definition:
+   1. Start VS Code and open your project folder.
+   2. Press <kbd>F1</kbd> select and **Remote-Containers: Add Development Container Configuration Files...** from the command palette.
+   3. Select the ReasonML definition.
+
+3. To use latest-and-greatest copy of this definition from the repository:
+   1. Clone this repository.
+   2. Copy the contents of `containers/reasonml/.devcontainer` to the root of your project folder.
+   3. Start VS Code and open your project folder.
+
+4. After following step 2 or 3, the contents of the `.devcontainer` folder in your project can be adapted to meet your needs.
+
+5. Finally, press <kbd>F1</kbd> and run **Remote-Containers: Reopen Folder in Container** to start using the definition.
+
+## Testing the definition
+
+This definition includes some test code that will help you verify it is working as expected on your system. Follow these steps:
+
+1. If this is your first time using a development container, please follow the [getting started steps](https://aka.ms/vscode-remote/containers/getting-started) to set up your machine.
+2. Clone this repository.
+3. Start VS Code, press <kbd>F1</kbd>, and select **Remote-Containers: Open Folder in Container...**
+4. Select the `containers/reasonml` folder.
+5. After the folder has opened in the container, use the following commands:
+
+```bash
+   cd test-project
+   yarn build # or npm run build, for npm
```

Comment:
I had to run yarn install first.  I assume that is expected?  If so, probably should add it here.